### PR TITLE
docs: ddl: document the initial_tablets replication strategy option

### PR DESCRIPTION
**Pull request by avikivity**
While the feature is experimental, this makes it easier to experiment with it.

### CODE REVIEW

**Comment by tzach** [comment]
An example would be helpful

```
CREATE KEYSPACE excalibur
    WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor' : 3, 'initial_tablets': 2048}
```

**Comment by avikivity** [comment]
v3: added an example

File: docs/cql/ddl.rst [hunk 1]
```diff
@@ -198,6 +200,34 @@ An example that excludes a datacenter while using ``replication_factor``::
...
+
+The experimental ``initial_tablets`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``initial_tablets`` option is used to specify how many tablets the table is split into.
+It is only valid when ``experimental_features: tablets`` is specified in ``scylla.yaml`` (which
+in turn requires ``consistent_cluster_management: true``. It must be a power of two.
+
+A good rule of thumb to calculate initial_tablets is to divide the expected total storage used
+by tables in this keyspace by (``replication_factor`` * 5GB). For example, if you expect a 30TB
```
Comment (tgrabiec) [inline docs/cql/ddl.rst:213]: One could think that he should sum the storage of all tables, so maybe clarify that he should take the storage of the largest table. And clarify that the option controls initial tablets for every table created in the keyspace.

**Review by tgrabiec — CHANGES_REQUESTED** [review]

File: docs/cql/ddl.rst (hunk 1 above)
Comment (avikivity) [reply docs/cql/ddl.rst:213]: yes; but we should eventually make it a per-table option.

**Comment by tgrabiec** [comment]
Conflicts, please rebase

**Review by tgrabiec — APPROVED** [review]

**Comment by bhalevy** [comment]
please rebase

**Comment by avikivity** [comment]
> 
> > Why is it only opensource? Why would we tag it here?
> 
> The feature is not opensource as such, but it's experimental (for now). We exclude experimental features from the Enterprise docs, as we claim that Enterprise does not support experimental features. That's the only reason.
> 


We can't edit each piece of documentation separately like that. If you want to exclude experimental features from enterprise, introduce markup to denote experimental features, and post-process the documentation to exclude them.

**Comment by avikivity** [comment]
v5: rebased

**Comment by avikivity** [comment]
@tgrabiec ping

**Pull request closed without merging**